src/models/position_inputs.h

```diff
@@ -16,6 +16,7 @@ struct DefaultPositionInputs : PositionInputs {
   void Update(DeviceSpan<int32_t> next_tokens, int total_length, int new_length) override;
 
   void RewindTo(size_t index) override;
+  std::string name_;
```

Comment:
Since PositionInputs implements logic for both position_ids and attention_mask, shall we be explicit about what name_ refers to?
```suggestion
  std::string attention_mask_name_;
```